pkg/webservice/handlers_test.go

```diff
@@ -468,7 +468,7 @@ func TestGetClusterUtilJSON(t *testing.T) {
 	assert.Equal(t, partitionName, partition.Name)
 	// new app to partition
 	appID := "appID-1"
-	app := newApplication(appID, partitionName, queueName, rmID, security.UserGroup{})
+	app := newApplication(appID, partitionName, queueName, rmID, security.UserGroup{User: "testuser", Groups: []string{"testgroup"}})
```

Comment:
`newApplication` is a wrapper defined in this file. A simpler change would be to check if and empty `security.UserGroup{}` is passed in in that implementation and then override with a test user.